### PR TITLE
Fixes #19502: Missing 'settings' layer in the allowed_networks API doc

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/19502

### CODE REVIEW

**Comment by Fdall** [comment]
Commit modified

File: webapp/sources/api-doc/openapi.src.yml [hunk 1]
```diff
@@ -143,11 +143,11 @@ paths:
     $ref: paths/system/info.yml
   "/system/reload":
     $ref: paths/system/reload.yml
-  "/reload/techniques":
+  "system/reload/techniques":
```
Comment (amousset) [inline webapp/sources/api-doc/openapi.src.yml:146]:
```suggestion
  "/system/reload/techniques":
```

File: webapp/sources/api-doc/openapi.src.yml [hunk 2]
```diff
@@ -143,11 +143,11 @@ paths:
...
     $ref: paths/system/reload-techniques.yml
-  "/reload/groups":
+  "system/reload/groups":
```
Comment (amousset) [inline webapp/sources/api-doc/openapi.src.yml:148]:
```suggestion
  "/system/reload/groups":
```

**Comment by Fdall** [comment]
PR rebased

**Review by fanf — APPROVED** [review]

**Comment by Fdall** [comment]
OK, squash merging this PR

**Comment by Fdall** [comment]
OK, merging this PR

**Pull request merged**